bemani/frontend/museca/endpoints.py

```diff
@@ -101,7 +101,7 @@ def viewnetworkrecords() -> Response:
     frontend = MusecaFrontend(g.data, g.config, g.cache)
     network_records = frontend.get_network_records()
     versions = {version: name for (game, version, name) in frontend.all_games()}
-    versions[0] = 'CS and Licenses'
```

Comment:
LOL copypasta from Pop'n. I wonder if there was a reason for it being here instead of in the `all_games` method. I should have wrote a comment...